.github/workflows/check_workflow.yaml

```
@@ -0,0 +1,16 @@
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: Setup Gradle
+        uses: gradle/gradle-build-action@v2
+
+      - name: Change Permission
+        run: chmod +x ./gradlew
+
+      - name: Execute Gradle check
+        run: ./gradlew ktlintCheck
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)